mmv1/third_party/terraform/services/container/resource_container_cluster.go.tmpl

```diff
@@ -2388,6 +2508,7 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 			DnsConfig:                            expandDnsConfig(d.Get("dns_config")),
 			GatewayApiConfig:                     expandGatewayApiConfig(d.Get("gateway_api_config")),
 			EnableMultiNetworking:                d.Get("enable_multi_networking").(bool),
+			DefaultEnablePrivateNodes:		      expandDefaultEnablePrivateNodes(d),
```

Comment:
```suggestion
			EnableMultiNetworking:                d.Get("enable_multi_networking").(bool),
			DefaultEnablePrivateNodes:            expandDefaultEnablePrivateNodes(d),
```

tabs -> spaces (not sure if I got it to line up quite right...)

Reply:
Done.